papis/git.py

```diff
@@ -57,10 +36,8 @@ def remove(path: str, resource: str, recursive: bool = False) -> None:
     :param resource: Commit resource
     """
     logger.info("Removing '%s'.", path)
-    # force removal always
```

Comment:
Was there a reason why `-f` was used initially? Does it break something to do a soft remove?

Reply:
I'm not sure, but I added it back in the updated version to not change the behavior.